labs/dxp/7.x/installation-and-upgrades/02-setting-up-liferay-dxp/installing-and-managing-apps/managing-apps.md

```diff
@@ -0,0 +1,58 @@
+# Managing Apps
+
+The App Manager is a Control Panel screen where you can install and uninstall apps, and examine installed apps, modules, and OSGi components.
```

Comment:
Suggested improvement to this sentence, to include activation/deactivation and read more smoothly.

|  The App Manager installs and uninstalls apps, activates and deactivates apps, and lists details about apps and their modules and OSGi components.